lib/game.rb

```diff
@@ -1,21 +1,40 @@
 class Game
-
   attr_reader :game_id, :season, 
               :type, :date_time, 
               :away_team_id, :home_team_id, 
               :away_goals, :home_goals, 
               :venue, :venue_link
   
   def initialize(game_data)
-    @game_id = game_data[:game_id]
-    @season = game_data[:season]
+    @game_id = game_data[:game_id].to_i
+    @season = game_data[:season].to_i
     @type = game_data[:type]
-    @date_time = game_data[:date_time]
-    @away_team_id = game_data[:away_team_id]
-    @home_team_id = game_data[:home_team_id]
-    @away_goals = game_data[:away_goals]
-    @home_goals = game_data[:home_goals]
+    @date_time = game_data[:date_time].to_i
+    @away_team_id = game_data[:away_team_id].to_i
+    @home_team_id = game_data[:home_team_id].to_i
+    @away_goals = game_data[:away_goals].to_i
+    @home_goals = game_data[:home_goals].to_i
     @venue = game_data[:venue]
     @venue_link = game_data[:venue_link]
   end
+
+  def self.all_games
```

Comment:
Nice I like this method, instead of having to call GameFactory.all_games a bunch

Reply:
yeah nice abstraction!